subgraph-api/src/subgraph.ts

```diff
@@ -0,0 +1,401 @@
+// Copied from webapp/utils/subgraph.ts and slightly modified
```

Comment:
Perhaps you may want to link here using the permalink, because once this is merged, that file will most likely be removed, and this comment won't make sense anymore